oaknational/Oak-Web-Application
# Feat/1225 download specific curriculum

## Description

- Download button added to specific curriculum on units page
- Curriculum download button added to lesson listing page

## Issue(s)

Fixes #1225 

## How to test

1. Go to https://deploy-preview-1345--oak-web-application.netlify.thenational.academy/beta/teachers/key-stages/ks4/subjects/maths/units?tier=foundation
2. Click on download links
3. You should see a download of the specified curriculum or zip file for ks4 maths

## Screenshots

How it used to look (delete if n/a):

<img width="762" alt="Screenshot 2023-03-01 at 14 34 46" src="https://user-images.githubusercontent.com/91190841/222171690-3034f3cf-add5-4042-bc47-a00838adaff6.png">


<img width="1487" alt="Screenshot 2023-03-01 at 14 34 57" src="https://user-images.githubusercontent.com/91190841/222171652-37334c14-4ff0-459f-a150-0ca93b09b6da.png">


How it should now look:
<img width="1492" alt="Screenshot 2023-03-01 at 14 33 51" src="https://user-images.githubusercontent.com/91190841/222171738-04521521-50d2-4674-bbd1-3052d5c5aa0e.png">

<img width="1493" alt="Screenshot 2023-03-01 at 14 35 19" src="https://user-images.githubusercontent.com/91190841/222171829-00b07507-67da-4b0a-884b-09c24e97d144.png">


## Checklist
